src/class_singleWorker.py

```diff
@@ -390,383 +452,623 @@ def sendOutOrStoreMyV4Pubkey(self, myAddress):
+                            queues.UISignalQueue.put((
+                                'updateSentItemStatusByAckdata', (
+                                    ackdata,
+                                    tr._translate(
+                                        "MainWindow",
+                                        "Problem: The work demanded by"
+                                        " the recipient (%1 and %2) is"
+                                        " more difficult than you are"
+                                        " willing to do. %3"
+                                    ).arg(str(float(
+                                        requiredAverageProofOfWorkNonceTrialsPerByte)
+                                        / defaults.networkDefaultProofOfWorkNonceTrialsPerByte
+                                        )).arg(str(float(
+                                            requiredPayloadLengthExtraBytes)
+                                            / defaults.networkDefaultPayloadLengthExtraBytes
+                                            )).arg(l10n.formatTimestamp()))
+                            ))
                             continue
-            else: # if we are sending a message to ourselves or a chan..
+            else:  # if we are sending a message to ourselves or a chan..
                 logger.info('Sending a message.')
-                logger.debug('First 150 characters of message: ' + repr(message[:150]))
+                logger.debug(
+                    'First 150 characters of message: %r', message[:150])
                 behaviorBitfield = protocol.getBitfield(fromaddress)
 
                 try:
                     privEncryptionKeyBase58 = BMConfigParser().get(
                         toaddress, 'privencryptionkey')
                 except Exception as err:
-                    queues.UISignalQueue.put(('updateSentItemStatusByAckdata',(ackdata,tr._translate("MainWindow",'Problem: You are trying to send a message to yourself or a chan but your encryption key could not be found in the keys.dat file. Could not encrypt message. %1').arg(l10n.formatTimestamp()))))
-                    logger.error('Error within sendMsg. Could not read the keys from the keys.dat file for our own address. %s\n' % err)
+                    queues.UISignalQueue.put((
+                        'updateSentItemStatusByAckdata', (
+                            ackdata,
+                            tr._translate(
+                                "MainWindow",
+                                "Problem: You are trying to send a"
+                                " message to yourself or a chan but your"
+                                " encryption key could not be found in"
+                                " the keys.dat file. Could not encrypt"
+                                " message. %1"
+                            ).arg(l10n.formatTimestamp()))
+                    ))  # log or show the address maybe?
```

Comment:
Well, this changes the status, so the recipient address is in the same row of the message list.